### ISSUE TITLE
No option to un-stack windows

### ISSUE THREAD

**Issue by TurkeyMan** [jake-phy/WindowIconList#118]
I've noticed with latest dev, that all windows of the same type are always combined into one button, and there seems to be no option to control this.
I don't think it was like that before...?
My preferred behaviour would be like Win7, which is that that each window of an app have its own task button within a connected cluster (they all move together), unless the taskbar is full; only then it would stack them. I don't like them stacked when they don't need to be, it just means one extra action (hover-pop-up and choose) to get to the window I want.

**Comment by jake-phy** [comment]
that is correct.  i am splitting out the part that stacks only when needed.  I'm going to rebase that off of the cinnamon window list, add the appmenu and replace the preview with mine.  It made this applet faster to split them out into separate applets.  Windows 7's default behavior is to always stack and I would like to make this applet focused toward that behavior.  Hopefully i'll have the other applet up this weekend.

**Comment by jake-phy** [comment]
Well that failed.  :/ the option to unstack was harder then I thought it would be to split out, so I have integrated it back in.
